device-types/Datacom/DM4050-24GT-6XS.yaml

```diff
@@ -0,0 +1,94 @@
+---
+manufacturer: Datacom
+model: DM4050 24GT+6XS
+slug: dm4050-24gt-6xs
+part_number: 800.5189.xx
+u_height: 1.0
+is_full_depth: true
+airflow: side-to-rear
+comments: L2/L3 wirespeed switch with 24 electrical Gigabit Ethernet ports (RJ45SFP) and 6 optical 10Gisgabit Ethernet ports (SFP+) 1U Metal cabinet for
+  19-inch racks. Two slots for power supplies Power supplies and SFPs to be ordered separately (https://www.datacom.com.br/uploads/product/file/97aede61eba553f23060d983b6e68152.pdf)
+weight: 2.96
+weight_unit: kg
+console-ports:
+  - name: Console
+    type: rj-45
+  - name: Console2
+    type: usb-mini-b
```

Comment:
Is this console port actually used for console access? If not, please remove it.